documentation/examples/custom-sd/adapter/adapter.go

```diff
@@ -18,16 +18,16 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"reflect"
-
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/prometheus/common/model"
 	"github.com/prometheus/prometheus/discovery"
 	"github.com/prometheus/prometheus/discovery/targetgroup"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"sort"
```

Comment:
We separate imports into 3 groups: stdlib, 3rd-party packages and local packages. These lines need to go back after the `"fmt"` line.

Reply:
you need an empty line after `"sort"`

Reply:
done